src/Optimization/hiopAlgPrimalDecomp.cpp

```diff
@@ -378,15 +384,15 @@ update_ratio(const double base_v, const double base_vm1)
   //printf("recourse estimate inside HessianApprox %18.12e\n",rk);
   double rho_k = (base_vm1+fkm1-fk-base_v)/(fkm1+base_vm1-rk-base_v);
    
-  if(ver_ >=outlevel2) {
-    printf("previuos base  %18.12e, current base %18.12e, previuos val  %18.12e, real val %18.12e, predicted val %18.12e, rho_k %18.12e\n",base_vm1,base_v,fkm1,fk,rk,rho_k);
-  }
+  log_->printf(hovSummary,"previuos base  %18.12e, current base %18.12e, previuos val  %18.12e," 
+               " real val %18.12e, predicted val %18.12e, rho_k %18.12e\n",base_vm1,base_v,fkm1,fk,rk,rho_k);
```

Comment:
typos -> "previuos"

enforce coding style-guide, for example, one arg per line (this will also take care of the missing spaces between variables).